org.gabmus.giara.json

```diff
@@ -0,0 +1,608 @@
+{
+    "app-id": "org.gabmus.giara",
+    "command": "giara",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.38",
+    "sdk": "org.gnome.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home:ro",
```

Comment:
What's this needed for?

Reply:
The user can make posts with an attached media like an image or a video.

Reply:
Does your app use `GtkFileChooserNative`? Then you don't need this permission, because the `xdg-desktop-portal` would give you safe access to all files.